test/fixtures/iotFleetProvisioning/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+// NOTE: the `utils.js` file is bundled into the deployment package
+// eslint-disable-next-line
+const { log } = require('./utils');
```

Comment:
Let's remove utils dependency.  This util adds markers which were used in some integration tests, but even over there this feature is deprecated.

We can use regular `console.log`. I've also pushed to master eslint config update, which allows to use `console.log` in fixtures